Bump amethyst version to 0.13.2

fkarg opened the pull request:
So I wanted to use the PrefabRegistry, but noticed that it was not compatible with the newer amethyst version (at first), and now I somehow ended up changing (and learning) a few things.

Anyhow, for some reason it is not working though. I only get a black screen. Also not sure how to further debug this.

Can someone give me some pointers about this or make the necessary adjustments? Thanks!

fkarg committed:
Upgrade to Amethyst v0.13.2

- Upgraded amethyst to v0.13.2
- Renamed Resources to World according to specs migration guide
- GltfSceneLoader became a SystemDesc (`with_system_desc`)
- PrefabLoader became a SystemDesc (`with_system_desc`)
- Use `world.insert` instead of `world.add_resource`
- Add UiGlyphSystemDesc as UiGlyphs were missing at runtime
- Several smaller search/replace changes

fkarg committed:
Fixed Typo (Resorces) and s/res/world

fkarg committed:
rustfmt

erlend-sh commented:
Thanks so much for having a go at an update pass!

The project is without any active maintainers right now, but maybe @marot, @sunreef or @khskarl can keep you going with some of those debug tips you're asking for.

fkarg committed:
Import (missing?) Builder trait. cargo check works

fkarg committed:
added commit about why the 'unused import' is there

fkarg committed:
cargo fmt

fkarg committed:
Fixed Jenkins (?)

- Removed Builder from module import
- Added to Test import only
- Removed unnecessary comment

ashneverdawn commented:
amethyst has been updated to 0.15.0
